app/src/main/java/org/kiwix/kiwixmobile/KiwixMobileActivity.java

```diff
@@ -733,6 +774,31 @@ private KiwixWebView newTab(String url) {
     tabDrawerAdapter.notifyDataSetChanged();
     setUpWebView();
     documentParser.initInterface(webView);
+    WebSettings webSettings = getCurrentWebView().getSettings();
+    //Setting text-size and the corrosponding seekbar progress.
+    switch (settings.getInt("text_size",2)) {
```

Comment:
Spacing... Please fix this in multiple locations. Commas must have a space after them. It is really annoying having to point each one of these issues out so please check the whole file.